storybook/stories/Examples/cartesian/Bar/WithBrushAndOnDragEnd.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Brush, ResponsiveContainer, Bar, BarChart, XAxis, YAxis } from '../../../../../src';
+import { dateWithValueData } from '../../../data';
+
+export default {
+  title: 'Examples/cartesian/Bar/With Brush and onDragEnd',
+};
+
+interface BrushStartEndIndex {
+  startIndex?: number;
+  endIndex?: number;
+}
+
+export const WithBrushAndOnDragEnd = {
+  render: () => {
+    const [dragIndexes, setDragIndexes] = React.useState<BrushStartEndIndex>({
+      startIndex: 0,
+      endIndex: dateWithValueData.length - 1,
+    });
+    return (
+      <div style={{ width: '100%', height: 'calc(100% - 84px)' }}>
+        <div>
+          Start index:
+          {dragIndexes.startIndex}
+        </div>
+        <div>
+          End index:
+          {dragIndexes.endIndex}
+        </div>
+        <ResponsiveContainer>
+          <BarChart data={dateWithValueData}>
+            <XAxis dataKey="value" />
+            <YAxis />
+            <Brush
+              dataKey="name"
+              height={30}
+              stroke="#8884d8"
+              onDragEnd={indexes => {
+                setDragIndexes(indexes as BrushStartEndIndex);
+              }}
+            />
+            <Bar dataKey="value" fill="#8884d8" />
```

Comment:
A tiny nit: I would prefer examples to be as simple as possible, i.e. really only containing the modifications that are required to showcase the feature - to increase salience of the example.

Thus I would suggest to simplify this a bit, i.e. remove the fill, remove the stroke of the brush, etc. Whatever can go, should. 

But also: Thank you for adding this example! :heart:

Reply:
No problem :) This has now be done. Also added a comment regarding the styles above. Let me know if that's too much.